db/test/helper.js

```diff
@@ -406,11 +412,15 @@ exports.testEntityTable = ({
         await exports.upgradeTo(dbVersion);
       });
 
-      for (let {condition, expectedSample} of loadConditions) {
-        test(`load ${JSON.stringify(condition)}`, async function() {
-          const entity = await Entity.load(condition);
-          assert.deepEqual(entity._properties, samples[expectedSample]);
-        });
+      if (notImplemented.includes('entity-functions')) {
+        test('database upgrade', async function() {});
```

Comment:
Why is this called `named database upgrade` when the test is empty? Would it make more sense to simply have the if statement without the `else`?

```
if (!notImplemented.includes('entity-functions')) {
  // ...
}
```

Reply:
I did this so that the test setup would be executed. Without a test, I think the test setup steps would never run, so this ensures at least one test is generated so that the test setup steps are executed as they are useful for testing the database scripts too even if data isn't migrated.

Reply:
Ah, I see. That makes sense. Thanks.